internal/pkg/cli/deploy/static_site.go

```diff
@@ -43,13 +48,40 @@ func NewStaticSiteDeployer(in *WorkloadDeployerInput) (*staticSiteDeployer, erro
 	}, nil
 }
 
+func staticSiteCustomResources(fs template.Reader) ([]*customresource.CustomResource, error) {
+	crs, err := customresource.StaticSite(fs)
+	if err != nil {
+		return nil, fmt.Errorf("read custom resources for a %q: %w", manifestinfo.StaticSiteType, err)
+	}
+	return crs, nil
+}
+
 // IsServiceAvailableInRegion checks if service type exist in the given region.
 func (*staticSiteDeployer) IsServiceAvailableInRegion(region string) (bool, error) {
 	return partitions.IsAvailableInRegion(s3.EndpointsID, region)
 }
 
+// GenerateCloudFormationTemplate generates a CloudFormation template and parameters for a workload.
+func (d *staticSiteDeployer) GenerateCloudFormationTemplate(in *GenerateCloudFormationTemplateInput) (
+	*GenerateCloudFormationTemplateOutput, error) {
+	conf, err := d.stackConfiguration(&in.StackRuntimeConfiguration)
+	if err != nil {
+		return nil, err
+	}
+	return d.generateCloudFormationTemplate(conf)
+}
+
 // DeployWorkload deploys a static site service using CloudFormation.
 func (d *staticSiteDeployer) DeployWorkload(in *DeployWorkloadInput) (ActionRecommender, error) {
+	conf, err := d.stackConfiguration(&in.StackRuntimeConfiguration)
+	if err != nil {
+		return nil, err
+	}
+	if err := d.deploy(in.Options, svcStackConfigurationOutput{
+		conf: cloudformation.WrapWithTemplateOverrider(conf, d.overrider),
+	}); err != nil {
+		return nil, err
+	}
 	return nil, nil
```

Comment:
The first param is always a `nil`. I guess this is to satisfy an interface 🤔 
What about returning a no-op `ActionRecommender` instead of `nil`?